cli/cmd/encore/app/create_form.go

```diff
@@ -0,0 +1,361 @@
+package app
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/charmbracelet/bubbles/list"
+	"github.com/charmbracelet/bubbles/spinner"
+	"github.com/charmbracelet/bubbles/textinput"
+	tea "github.com/charmbracelet/bubbletea"
+	"github.com/charmbracelet/lipgloss"
+	"github.com/tailscale/hujson"
+
+	"encr.dev/cli/cmd/encore/cmdutil"
+)
+
+const (
+	codeBlue   = "#6D89FF"
+	codePurple = "#A36C8C"
+	codeGreen  = "#B3D77E"
+)
+
+var (
+	inputStyle = lipgloss.NewStyle().Foreground(lipgloss.AdaptiveColor{Dark: codeBlue, Light: codeBlue})
+	descStyle  = lipgloss.NewStyle().Foreground(lipgloss.AdaptiveColor{Dark: codeGreen, Light: codePurple})
+	docStyle   = lipgloss.NewStyle().Margin(1, 2, 0, 2)
+)
+
+type item struct {
+	ItemTitle string `json:"title"`
+	Desc      string `json:"desc"`
+	Template  string `json:"template"`
+}
+
+func (i item) Title() string       { return i.ItemTitle }
+func (i item) Description() string { return i.Desc }
+func (i item) FilterValue() string { return i.ItemTitle }
+
+type model struct {
```

Comment:
I think this should probably be renamed, to allow us to start to use bubbletea on other UI parts within the app later on.


```suggestion
type createFormModel struct {
```